_env.platform.zsh

```diff
@@ -21,13 +29,33 @@ export CRED_TOKEN=$(op read "op://Employee/$ONE_PASSWORD_ITEM/CRED_TOKEN")
 export ASTER_URL=${ASTER_LOGIN_URL}
 export ASTER_USER_EMAIL=${ASTER_EMAIL}
 export ASTER_USER_PASSWORD=${ASTER_EMAIL_PWD}
-export FIREBASE_API_KEY=$(op read "op://Employee/$ONE_PASSWORD_ITEM/firebase-api-key") 
+
+if FIREBASE_API_KEY=$(op read "op://Employee/$ONE_PASSWORD_ITEM/firebase-api-key"); then
+  export FIREBASE_API_KEY
+elif FIREBASE_API_KEY=$(op read "op://Cloud Development/Firebase/LOCAL/Api key"); then
+  echo "Using FIREBASE_API_KEY from op://Cloud Development/Firebase/LOCAL/Api key"
```

Comment:
I was not aware of this items in 1password!
I think that then they should be the single source of truth, and, just as we set `ONE_PASSWORD_ITEM` before sourcing `_env.platform.sh`, we should be setting `ONE_PASSWORD_ITEM_FIREBASE`, or similar.

Reply:
Here I was thinking that as default we should use `op://Cloud Development/Firebase/LOCAL/Api key` but still maybe it would be good to have the flexibility to provide FIREBASE_API_KEY, but not sure if this is ever a valid scenario